partner_manual_rank/README.rst

```diff
@@ -63,6 +63,7 @@ Contributors
 -  Francesco Apruzzese <cescoap@gmail.com>
 -  Francisco Luna <fluna@vauxoo.com>
 -  Luis González <lgonzalez@vauxoo.com>
+-  Le Dinh Tien <tien-ld@komit-consulting.com>
```

Comment:
Could you add yourself only in the file `CONTRIBUTORS.md`, please?

The rest (this README and the index files) will be taken care of by the bot.

Reply:
Hi @luisg123v, during the migration progress, I only added myself in `CONTRIBUTORS.md`
the README and other files were updated by pre-commit

So can we include it in this commit? or just ignore these changes and update `CONTRIBUTORS.md` only?

Reply:
AFAIK we should modify only `CONTRIBUTORS.md`, and the rest will be updated automatically after migration. This so that those changes are not included as part of the migration commit.

Could you try that, please?

Reply:
yes of course, I updated the commit @luisg123v